package.json

```diff
@@ -12,7 +12,8 @@
     "redux": "^3.6.0",
     "redux-devtools-extension": "^1.0.0",
     "redux-logger": "^2.7.4",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",
+    "zoo-grommet": "^0.1.0"
   },
```

Comment:
I'm really regretting this now. For a button and a modal, I need to change 5 dependencies 👎